native/cocos/platform/android/java/src/com/cocos/lib/CocosDownloader.java

```diff
@@ -136,43 +134,27 @@ public void run() {
                 Call task = null;
 
                 do {
-                    if (path.length() > 0) {
-                        try {
-                            URI uri = new URI(url);
-                            domain = uri.getHost();
-                        } catch (URISyntaxException e) {
-                            e.printStackTrace();
-                            break;
-                        } catch (NullPointerException e) {
-                            e.printStackTrace();
-                            break;
-                        }
-
+                    if (pathWithUrlHash.length() > 0) {
```

Comment:
`pathWithUrlHash` length is always non-zero, and the behavior has changed

Reply:
change to previoius logic